easybuild/easyconfigs/m/MOSAIK/MOSAIK-2.2.28-goolf-1.4.10_2014-04-25.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'MOSAIK'
+version = '2.2.28'
+git_commit_id = '24cf06' #first six digits of the commit id we will download from github
+versionsuffix = '-20140425-%s' % git_commit_id
+
+homepage = 'https://code.google.com/p/mosaik-aligner/'
+description = """ MOSAIK is a reference-guided aligner for next-generation 
+ sequencing technologies """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://github.com/wanpinglee/MOSAIK/archive/']
+sources = ['%s.tar.gz' % (git_commit_id)]
+
+dependencies = [('zlib', '1.2.8')]
+
+parallel = 1
+
+start_dir = "src"
+
+makeopts = ' BIN_DIR="./bin" OBJ_DIR="./obj"'
+
+files_to_copy = ["bin", "../README", "demo"]
+        
+sanity_check_paths = {
+    'files': ["bin/%s" % x for x in ["MosaikAligner", "MosaikBuild", "MosaikJump", "MosaikText"]],
+    'dirs': [],
+}
+
+moduleclass = 'bio'
+
```

Comment:
trim empty lines at end of files: this will trigger a more recent jenkins test, which is good.
minor changes needed here, put this in REVIEW plz!